### PR TITLE
Rename static methods for constants to `const_<name>` and add `Board.size` instance method

### PR DESCRIPTION
**Pull request by Yo1k**

### CODE REVIEW

**Comment by Yo1k** [comment]
I did not know `github` anyway formats the commit name in this way. 
I will rename this commit to "Rename constants to `const_<name>` and add `Board.size` instance method".

**Review by stIncMale — APPROVED** [review]
Congrats with the first PR that was approved right away :ъ

File: yo1k/tic_tac_toe/test/core/test_game.py [hunk 1]
```diff
@@ -214,7 +210,7 @@ def test_win(self) -> None:
                 None,
                 Action.new_occupy(Cell(0, 1))])
         logic = Logic((act_queue_px, act_queue_po))
-        for _ in range(0, len(act_queue_px.actions) + len(act_queue_po.actions)):
+        while len(act_queue_px.actions) + len(act_queue_po.actions) != 0:
```
Comment (stIncMale) [inline yo1k/tic_tac_toe/test/core/test_game.py]: While `!= 0` is equivalent to `> 0` in this particular case, the latter conveys more info: while we are iterating the left side is non-negative and we are going to iterate until it reaches 0. `!= 0` does not convey the part _"while we are iterating the left side is non-negative"_. One can derive this part by actually looking at what we have in the left side, but that requires giving it a little bit more thought.

**Review by stIncMale — CHANGES_REQUESTED** [review]

File: yo1k/tic_tac_toe/test/core/test_game.py (hunk 1 above)
Comment (Yo1k) [reply yo1k/tic_tac_toe/test/core/test_game.py]: Done.

**Review by stIncMale — APPROVED** [review]

**Pull request merged**